applications/app/views/offlinePage.scala.html

```diff
@@ -14,10 +17,65 @@
     @* get the stylesheets downloading ASAP *@
     @fragments.stylesheets(None)
 
-    @* load the app *@
+    @* We don't use the inlineJSBlocking template because we need to specify
+    our own paths *@
```

Comment:
For context: back when I added the offline page, @rich-nguyen and I decided not to re-use the main template, but just do extract the things needed for the offline page. This gives us increased flexibility, allowing us to specify custom paths (see below). We need to set custom paths because we must use bundles in dev, so that the offline page can be tested.